optuna/batch/study.py

```diff
@@ -0,0 +1,197 @@
+import copy
+from functools import partial
+import gc
+import math
+import types
+from typing import Any
+from typing import Callable
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Type
+from typing import Union
+
+import numpy as np
+
+import optuna
+from optuna._experimental import experimental
+import optuna.logging
+
+ObjectiveFuncType = Callable[["optuna.batch.trial.BatchTrial"], np.ndarray]
+CallbackFuncType = Callable[["optuna.study.Study", "optuna.trial.FrozenTrial"], None]
+
+_logger = optuna.logging.get_logger(__name__)
+
+
+class BatchStudy(object):
```

Comment:
With MO, we also marked classes experimental in addition to `create_study` and `load_study`. How about following the same convention?

Reply:
I added it in https://github.com/optuna/optuna/pull/1729/commits/6860311582ef170021ce26a7af2f3e77a6201a63. Thanks!